pkg/perf/tcp_task.go

```diff
@@ -0,0 +1,50 @@
+package perf
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/zos/pkg/network/iperf"
+)
+
+// TCPTask is the task for iperf tcp tests
+type TCPTask struct {
+	TaskID    string
+	Schedule  string
+	Bandwidth string
+	ClientIP  string
```

Comment:
These all need to be private fields (based on previous commet)

Also as explained, we need to run multiple tests against multiple nodes per run. hence the random list of nodes plust few fixed ones so u have like total of +10 nodes. then tests can run against these in sequence for bot tcp and udp, build and then return the report